src/quo2/screens/info/page_nav.cljs

```diff
@@ -0,0 +1,138 @@
+(ns quo2.screens.info.page-nav
+  (:require [quo.previews.preview :as preview]
+            [quo.react-native :as rn]
+            [quo.theme :as theme]
+            [quo2.components.info.page-nav :as quo2]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]))
+
+(def descriptor [{:label "Align middle-component to the left"
```

Comment:
the number of options in this example is quite extensive. Is there any way to simplify this at all?

It seems like the options could be 

1. -> mid section component - select with some preloaded examples 
-> text with page title
-> drop down menu with Recent text
-> Group avatar with title and description
-> channel name with icons
-> token component with name and icon
## using the same text etc as the Figma designs would make it really easy to verify here too.


2. align mid section - select with options
-> flex-start
-> centre

3. number of right icons
->1
->2
->3

Just my opinion on this. Perhaps it would be good to discuss with the UX team what options they like to have on these pages too.